easybuild/easyconfigs/g/git-annex/git-annex-10.20230802-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'MakeCp'
+
+name = 'git-annex'
+version = '10.20230802'
+
+homepage = 'https://git-annex.branchable.com'
+description = """git-annex allows managing large files with git, without storing the file contents in git. It can sync,
+backup, and archive your data, offline and online. Checksums and encryption keep your data safe and secure. Bring the
+power and distributed nature of git to bear on your large files with git-annex."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+dependencies = [
+    ('GHC', '9.4.6', '-x86_64', SYSTEM),
+    ('GCC', '12.3.0', '', SYSTEM),
```

Comment:
```suggestion
builddependencies = [('binutils', '2.40')]

dependencies = [
    ('GHC', '9.4.6', '-x86_64', SYSTEM),
```

Reply:
Yep, my fault. I updated Stack, anyway. I'll send over the changes